build/config/ios/rules.gni

```diff
@@ -115,14 +119,14 @@ template("ios_app") {
     script = ios_app_script
 
     sources = [ invoker.info_plist ]
-    outputs = [ "$root_build_dir/Info.plist" ]
+    outputs = [ "$root_build_dir/plist/$app_name/Info.plist" ]
```

Comment:
still not a big fan of adding a new top level build directory, but we can fix this in another pass.